### ISSUE TITLE
Example of custom file loader config lack of documentation

### ISSUE THREAD

**Issue by podarok** [salesforce/logai#57]
![image](https://user-images.githubusercontent.com/563412/236775084-c21792df-c655-4074-9573-607b65c0e2e6.png)

Any help in how to implement log analysis for syslog?

**Comment by Prateek12333** [comment]
Yes those steps have not been mentioned in the documentation. If you want your own custom file to get run for Log Summarization use Custom_YourFileName will bring that file in Log File filter for Log Summarization. For Anomaly Detection and Log Clustering I am also not sure and trying with trial and error to sort out the errors/exceptions.

**Comment by aadhineniganesh** [comment]
@Prateek12333 Any updates or any resources found to learn more about Anomaly Detection and Log Clustering with custom logs.

**Comment by csmitbhensdadia** [comment]
_**@aadhineniganesh**_ No updates and no learning resources found on Anomaly Detection and Log Clustering for now. Those algorithms are working because they have hardcoded values that are mentioned in examples for both algorithms. If we try our own custom file there is no sort of solution which we can implement for those algorithms . If anything comes up will let you know and bring up the solution here. But as per my RND work we can bring out custom file/logs for both the algorithms.

**Comment by Prateek12333** [comment]
_@aadhineniganesh_ No updates and no learning resources found on Anomaly Detection and Log Clustering for now. Those algorithms are working because they have hardcoded values(code that they have shared in GitHub link) in examples for both algorithms. If we try our own custom file there is no sort of solution which we can implement for those algorithms . If anything comes up will let you know and bring up the solution here. But as per my RND work we cannot bring out custom file/logs for both the algorithms.
